src/languages/en.js

```diff
@@ -680,6 +680,17 @@ export default {
             },
         },
     },
+    colorThemePage: {
```

Comment:
I think we should rename `ColorThemePage` to `themePage` to match the pattern of the other user preferences, what do you think?